frontend/src/components/VCheckbox/VCheckbox.vue

```diff
@@ -67,7 +68,7 @@ import VIcon from "~/components/VIcon/VIcon.vue"
 type CheckboxAttrs = {
   name: string
   value: string
-  disabled?: boolean
+  disabled?: "disabled" | ""
```

Comment:
Having to replace logical boolean with a string is unfortunate. This however wouldn't work because setting `disabled=""` still disables the control.

> To represent a false value, the attribute has to be omitted altogether.
> &mdash; [The spec](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attribute)